static/js/components/widgets/MarkdownEditor.tsx

```diff
@@ -53,6 +55,7 @@ export default function MarkdownEditor(props: Props): JSX.Element {
   const { link, embed, value, name, onChange, minimal, allowedHtml } = props
   const throwSynchronously = useThrowSynchronously()
   const website = useWebsite()
+  const { uuid } = useParams<{ uuid: string }>()
```

Comment:
The `uuid` isn't available for `useParams` in the context of a new `WebsiteContent` creation (eg. page). This means, if any ER is created or added to a page at the time of its creation, it won't be referenced currently.

As we address this, we should also ensure that it's a general purpose solution applicable for other `WebsiteContent` types as well where we may be able to use ERs.